src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ModuleBuilderImpl.cs

```diff
@@ -829,7 +832,7 @@ private MemberInfo GetOriginalMemberIfConstructedType(MemberInfo memberInfo)
             return memberInfo;
         }
 
-        private static Type[] ParameterTypes(ParameterInfo[] parameterInfos)
+        internal static Type[] ParameterTypes(ParameterInfo[] parameterInfos)
```

Comment:
Nit: Could you please move this method to MetadataSignatureHelper now that it is used outside this type?